packages/openapi-typescript/package.json

```
@@ -41,10 +41,10 @@
     "build": "del dist && tsc -p tsconfig.build.json",
     "dev": "tsc -p tsconfig.build.json --watch",
     "download:schemas": "vite-node ./scripts/download-schemas.ts",
-    "format": "pnpm run prettier -w .",
+    "format": "prettier --write 'src/**/*'",
     "lint": "run-p -s lint:*",
-    "lint:js": "eslint \"src/**/*.{js,ts}\"",
-    "lint:prettier": "prettier --check \"src/**/*\"",
+    "lint:js": "eslint 'src/**/*.{js,ts}'",
```

```suggestion
    "lint:js": "eslint \"src/**/*.{js,ts}\"",
```

We actually want to keep the escaped double quotes. Though *nix and mac users can run this just fine, IIRC Windows contributors run into problems with single quotes.

---

Since this is such a minor thing, I’ll revert this after merge, but won’t touch anything else. Thanks!

---

Sorry for that I always work on a UNIX OS and though the scripts without escaped quotes look nicer.

---

oh no worries at all. I did the same thing until I had Windows users complain 😛 (and I agree with you)

---

oh no worries at all. I did the same thing until I had Windows users complain 😛 (and I agree with you)